docroot/modules/custom/foia_webform/src/Plugin/QueueWorker/FoiaSubmissionQueueWorker.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Drupal\foia_webform\Plugin\QueueWorker;
+
+use Drupal\Core\Annotation\QueueWorker;
+
+/**
+ * Provides functionality for working with the queued FOIA form submissions.
+ *
+ * @QueueWorker (
+ *   id = "foia_submissions",
+ *   title = @Translation("Manual FOIA Form submitter"),
+ * )
+ */
+class FoiaSubmissionQueueWorker extends QueueWorker {}
```

Comment:
you had this right previously - must've been a typo, but this should extend `QueueWorkerBase`